0000-system-methods.md

```diff
@@ -0,0 +1,221 @@
+- Proposal Name: System methods in runtime API
+- Start Date: 2019-09-03
+- NEP PR: [nearprotocol/neps#0000](https://github.com/nearprotocol/neps/pull/0000)
+- Issue(s): link to relevant issues in relevant repos (not required).
+
+# Summary
+[summary]: #summary
+
+Adds new ability for contracts to perform some system functions:
+- create new accounts (with possible code deploy and initialization)
+- deploy new code (upgrades)
+- batched function calls
+- transfer money
+- stake
+- add key
+- delete key
+- delete account
+
+# Motivation
+[motivation]: #motivation
+
+Contracts should have the ability to create new accounts, transfer money without calling code and
+stake. It will enable full functionality of contract-based accounts.
+
+# Reference
+
+We introduce additional promise APIs to support batched actions.
```

Comment:
Somewhere in this NEP we should emphasize that the existing promise API is the special case of batched promise API and we should give a specific way to reproduce the existing promise API through this promise API.

Reply:
done